webpack_config/electron-main.development.js

```diff
@@ -127,10 +127,17 @@ module.exports = {
     })
   ],
 
-  externals: {
-    'node-hid': 'commonjs node-hid',
-    usb: 'commonjs usb'
-  },
+  externals: [
+    {
+      'node-hid': 'commonjs node-hid',
+      usb: 'commonjs usb'
+    },
+    {
+      // The following line removes the need to add the aws-sdk in the package.json
+      // https://medium.com/@sheepsteak/excluding-the-aws-sdk-from-a-serverless-package-8dcad2f31954
+      "aws-sdk": "commonjs aws-sdk"
```

Comment:
Not sure of the consequences of this one.
aws-sdk was added there for some reasons I don't understand
[](https://github.com/MyCryptoHQ/MyCrypto/pull/2441/commits)

Reply:
Also I've added `aws-sdk`.
Also I could have removed both `node-hid` and `usb` on the lines above.
The effect would have been the same.
Still I'm not sure about the consequences.